python/cuml/tests/dask/test_dask_logistic_regression.py

```diff
@@ -336,3 +344,60 @@ def test_noreg(fit_intercept, client):
     l1_strength, l2_strength = lr._get_qn_params()
     assert l1_strength == 0.0
     assert l2_strength == 0.0
+
+
+def test_n_classes_small(client):
```

Comment:
If we converted `test_n_classes` into a hypothesis test function, then the examples tested here could be listed as explicit examples.

Reply:
`test_n_classes_small` mainly tests the correctness of _num_classes (equivalently getUniqueLabelsMG). It covers corner cases, for example, 2 GPUs, one has label 0 and the other has label 1. Also, it covers the case where labels are non-continuous, such as [10., 20.]. It seems SG and MG fit, tested in `test_n_classes`, do not work for non-continuous labels yet.

Reply:
Ok and fine to leave it as-is, but I feel like the way the test is written we should either use hypothesis examples or pytest.parametrize.